components/dashboard/src/teams/Members.tsx

```diff
@@ -123,10 +156,10 @@ export default function() {
                         <span className="text-gray-400 flex-grow capitalize">{m.role}</span>
                         <ItemFieldContextMenu menuEntries={[
                             {
-                                title: 'Remove',
+                                title: (m.userId === user?.id) ? 'Leave Team' : 'Remove',
```

Comment:
I thought about it, but didn't have a good answer. We could also prompt "Hey, you're the last owner! Nominate someone else, or delete team?"

Reply:
Some of the possible solutions here:
1. Keep the _Leave_ option as is and on click trigger a toast notification with an error message explaining that you cannot leave because you are the only remaining owner of the team. Needs: https://github.com/gitpod-io/gitpod/issues/3530.
2. Keep the _Leave_ option as a disabled option (`text-red-300`) and show a tooltip on hover explaining why you cannot perform this action. See screenshot below.

| Disabled dropdown option | Disabled dropdown button |
|-|-|
| <img width="472" alt="Screenshot 2021-06-25 at 4 06 33 PM" src="https://user-images.githubusercontent.com/120486/123429186-53742800-d5cf-11eb-9982-66aaea1d3e83.png"> |  <img width="421" alt="Screenshot 2021-06-25 at 4 08 04 PM" src="https://user-images.githubusercontent.com/120486/123429393-9504d300-d5cf-11eb-937b-ac79baf51b0a.png"> |

Reply:
Follow-up issue: https://github.com/gitpod-io/gitpod/issues/4623